src/csharp/Microsoft.Spark/Sql/GenericRow.cs

```diff
@@ -0,0 +1,168 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Spark.Sql.Types;
+
+namespace Microsoft.Spark.Sql
+{
+    /// <summary>
+    /// Represents a row object in RDD, equivalent to GenericRow in Spark.
+    /// </summary>
+    public sealed class GenericRow
+    {
+        /// <summary>
+        /// Constructor for the Row class.
```

Comment:
Should it be `/// Constructor for the GenericRow class.`?

Reply:
Yes missed it, it should be GenericRow